tokenbucket.go

```diff
@@ -364,6 +364,12 @@ func (t *TokenBucketRedis) State(ctx context.Context) (TokenBucketState, error)
 	var values []interface{}
 	var err error
 	done := make(chan struct{}, 1)
+
+	if t.raceCheck {
+		// reset in a case of returning an empty TokenBucketState
+		t.lastVersion = 0
+	}
```

Comment:
Do you think it makes sense to add the test that covers this?

Reply:
@mennanov I can commit the test mentioned here: https://github.com/mennanov/limiters/issues/29. It is not so "generic" compared to other tests but the issue is not related to other backends.

Reply:
Hm.. let's keep the tests generic.
I will merge this PR as is.